data-collection/src/main/java/refactoringml/db/ClassMetric.java

```diff
@@ -137,6 +140,13 @@ public String toString() {
 				'}';
 	}
 
+	public boolean isSubclass(){ return isSubclass; }
+
+	/*
+	Only works with the class name from ck, because there subclasses are marked with a $ symbol.
+	 */
+	public static boolean evaluateSubclass(String className) { return className.contains("$"); }
```

Comment:
Neat!

CK also returns this information to you, via the `ck.getType()` method. If it's equals to "subclass", then it's a subclass. See https://github.com/mauricioaniche/ck/blob/master/src/main/java/com/github/mauricioaniche/ck/CKVisitor.java#L320

Which one do you think it's better to use?

Reply:
I think the CK one is better, because in cases there are $ symbols, for what ever reason within the class name, but not marking subclasses my solution would mark it incorrectly. Nonetheless, I would never expect a $ in the name.